bin/publish-swagger-docs.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
can't we curl this script in from somewhere rather than adding it to all the repos?

Reply:
we just had the same discussion/enlightenment. on it!